1-run-automation-test.md

```diff
@@ -0,0 +1,122 @@
+
+Set up your project and link your app repository that we will test with Kobiton later.
++ Instruction on how to integrate GitHub with TeamCity can be found in [here](https://confluence.jetbrains.com/display/TCD10/Integrating+TeamCity+with+VCS+Hosting+Services).
+ 
+>For more information about TeamCity can be found in [this tutorial](https://confluence.jetbrains.com/display/TCD10/TeamCity+Documentation).
+
+## 2. Config automation test script
+This documentation uses NodeJS as the default language to demonstrate the guide. 
+### 2.1 Get Username and API key
+Go to https://portal.kobiton.com.
+- Username
+  + Select *user icon* -> *Profile* (you might find it in the top right corner)
+
+  ![alt text](./assets/username.png )
+
+- API key 
+  + Select *user icon* -> *Settings* (you might find it in the top right corner)
+
+  ![alt text](./assets/api_key.png )
+
+### 2.2 Get desired capabilities
+The desired capabilities need to be added to the automation test script to allow the test to be executed on Kobiton device.
+  + Click *Devices* in the top navigation bar.
+ 
+    ![alt text](./assets/devices.png )
+ 
+  + Hover over any device you want to test with and click on the Automation settings button. A dialog box will then appear on screen.
+ 
+    ![alt text](./assets/click_auto.png )
+ 
+  + On the left hand side, you can select your preferred language, App Type, etc.
+
+    ![alt text](./assets/automation.png ) 
+### 2.3 Config TeamCity project
+Kobiton has already provided sample scripts for automation testing, visit [here](https://github.com/kobiton/samples) for reference.
+
+Create a Github repository for your script and configure with TeamCity.
+
+Follow steps below to setup TeamCity for automation testing.
++ In your Teamcity project, on the left side of the page, click *Build Step* to set build steps.
++ On the right side of the page, for the *Runner type* category, choose *Command Line*.
++ Add below commands to `Custom script` part.
+
+```
+  npm install
+  npm run android-app-test
+```
+> Note:
+> + `npm install` : install missing dependencies
+> + `npm run android-app-test` : run the test script
+
+![alt text](./assets/cmd.png)
+
+**(Optional) Attaching environment variable to Teamcity**
++ In your project in TeamCity, on the left side of the page, click *Parameters*. Then click `Add new parameter` button. 
+
+![alt text](./assets/param.png)
+
++ Add your Kobiton username and API key.
+
+![alt text](./assets/param_name.png)
+
+![alt text](./assets/param_key.png)
+### 2.4 Config automation test script
+Replace values of `username`, `apiKey` and `desiredCaps` in the script with the ones collected above.
+
+Example:
+
+```
+const username = {kobiton_username}
+const apiKey = {kobiton_api_key}
+ const kobitonServerConfig = {
+  protocol: 'https',
+  host: 'api-test.kobiton.com',
+  auth: `${username}:${apiKey}`
+}
+ var desiredCaps = {
```

Comment:
Please double check this desiredCaps with Device Name and platformVersion, platformName,...